ci-operator/step-registry/telcov10n/metal-single-node-hub-ztp/verify-hub/telcov10n-metal-single-node-hub-ztp-verify-hub-commands.sh

```
@@ -171,7 +171,8 @@ function test_hub_cluster_deployment {
   echo "************ telcov10n Test Hub deployment ************"
 
   set -x
-  diff -u ${KUBECONFIG} ${hub_kubeconfig} || \
+  # diff command is not available in the container using cmp instead
+  cmp -l ${KUBECONFIG} ${hub_kubeconfig} || \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)